packages/shared/wallet/src/connect-wallet-side-drawer/connect-wallet-side-drawer.tsx

```diff
@@ -2,55 +2,47 @@ import { useCallback } from 'react';
 import { Box, styled } from '@mui/material';
 import { LabelValue, SideDrawerActiveButton } from '@notional-finance/mui';
 import { FormattedMessage } from 'react-intl';
-import { useSideDrawerManager } from '@notional-finance/side-drawer';
 import { ViewAsAccount } from '../view-as-account/view-as-account';
-import { modules } from '../onboard-context';
-import { useConnect } from '../hooks/use-connect';
+import { useWalletConnect } from '../hooks/use-connect';
 import { useLocation } from 'react-router-dom';
-import {
-  useAccountReady,
-  useWalletConnectedNetwork,
-} from '@notional-finance/notionable-hooks';
+import { useAccount } from 'wagmi';
 
 export const ConnectWalletSideDrawer = () => {
-  const { connectWallet, currentLabel } = useConnect();
   const { pathname } = useLocation();
-  const network = useWalletConnectedNetwork();
-  const connected = useAccountReady(network);
-  const { clearWalletSideDrawer } = useSideDrawerManager();
+  const { connectWallet, displayConnectors } = useWalletConnect();
+  const { address } = useAccount();
```

Comment:
I'd prefer not to use any `wagmi` functions outside of the `use-connect` hook. Can we get the address from global state instead?